### ISSUE TITLE
[Feature request] Allow multiple rows in (grouped) window list applet

### ISSUE THREAD

**Issue by AndreiMiculita** [linuxmint/cinnamon#9746]
Hello, using a window list with window titles enabled, the buttons quickly become too small to display the complete titles. A feature like Windows XP/Vista/7's multi-row window list would be very useful (see image).

Could be implemented with a (configurable) max applet height. Once the panel exceeds that height, the applet splits horizontally to accommodate 2 rows. And so on, for more rows.

I wasn't sure whether to suggest this for the old window list applet, or the grouped window list applet. It seems that the latter can completely replace the former, with the right settings. Not sure why the old applet hasn't been retired completely.

<img src="https://nautilusmode.files.wordpress.com/2014/09/taskbar_81.png" alt="alt text" width="400">

**Comment by ItzSwirlz** [comment]
When the windows go over a row, the windows just squeeze, right?

Windows 7 doesn't have this feature if I recall, but it's something that can definitely be added (I'm not really that good with Cinnamon and JS yet unfortunately so I won't be able to help much)

**Comment by AndreiMiculita** [comment]
@ItzSwirlz Yes, when the buttons don't fit, they squeeze into place and you can't see the full title anymore.

I'm pretty sure the screenshot I provided is from Windows 7. In any case, here is a screenshot of it in Windows 10 as well. 
<img src="https://filestore.community.support.microsoft.com/api/images/aff509d6-232e-4043-b519-c6b57b0e38d0?upload=true" alt="alt text" width="400">

**Comment by StargazerA5** [comment]
I strongly support this feature for both the grouped and non-grouped Window lists.  It would also allow you to deprecate the very buggy Cinnamon Multi-Line Taskbar applet as this is the entire reason for it's existence.

**Comment by DuncanMarshall** [comment]
I want this too.

**Comment by StargazerA5** [comment]
There are still no good options for multi-line task windows lists....

**Comment by urikjaro** [comment]
Does anyone have some solution / alternative?
This is my taskbar and its quite annoying to navigate..
<img width="2555" height="53" alt="Image" src="https://github.com/user-attachments/assets/bf5630fb-6dda-43f3-938c-d56704a10aed" />

**Comment by JosephMcc** [comment]
Not planned.
